application/helpers/update/updatedb_helper.php

```diff
@@ -5050,6 +5050,23 @@ function ($v) {
             $oDB->createCommand()->update('{{settings_global}}', ['stg_value' => 479], "stg_name='DBVersion'");
             $oTransaction->commit();
         }
+        /**
+         * Add missing noTablesOnMobile.css to vanilla configs again. It was done on 428, but the bug on LsDefaultDataSets remained
+         * causing problems on new installations.
+         */
+        if ($iOldDBVersion < 480) {
+            $oTransaction = $oDB->beginTransaction();
+            // Update vanilla config
+            $oDB->createCommand()->update(
+                '{{template_configuration}}',
+                [
+                    'files_css' => '{"add":["css/base.css","css/theme.css","css/custom.css","css/noTablesOnMobile.css"]}',
+                ],
+                "template_name = 'vanilla' AND files_css != 'inherit'"
```

Comment:
`files_css like '{"add":["css/ajaxify.css","css/theme.css","css/custom.css"]}'` ?

Reply:
bootswatch and fruity don't have the same issue ?

They have the `css/ajaxify.css`

Reply:
> `files_css like '{"add":["css/ajaxify.css","css/theme.css","css/custom.css"]}'` ?

Alt : loop for all css, json_decode, add noTablesOnMobile if not inside, json_encode.